packages/types/src/request.ts

```diff
@@ -1,4 +1,7 @@
-/** Request data included in an event as sent to Sentry */
+/**
+ * Request data included in an event as sent to Sentry.
+ * TODO(v9): Rename this to avoid confusion, because Request is also a native type.
+ */
 export interface Request {
```

Comment:
l: Can't we change this now? What exposed to users relies on this?

Reply:
it is exported from `@sentry/types`, _probably_ does not affect anybody, but 🤷 IMHO I would just clean this up and rename these things in v9, WDYT?